addons/analytic/models/analytic_mixin.py

```diff
@@ -53,6 +54,9 @@ def _compute_distribution_analytic_account_ids(self):
             ids = list(unique(int(_id) for key in (rec.analytic_distribution or {}) for _id in key.split(',') if int(_id) in existing_accounts_ids))
             rec.distribution_analytic_account_ids = self.env['account.analytic.account'].browse(ids)
 
+    def _search_distribution_analytic_account_ids(self, operator, value):
+        return [('analytic_distribution', operator, value if isinstance(value, list) else [value])]
```

Comment:
What are the operator and value that require to create a list?

Wouldn't it make more sense to manage the cast to a list in `_condition_to_sql`?

Reply:
> What are the operator and value that require to create a list?

I just had the case for the `=` operator where value was the `id` of the account with the horizontal group use case.

> Wouldn't it make more sense to manage the cast to a list in `_condition_to_sql`?

Probably. I will check.

Thanks.